### PR TITLE
fix(select): add aria-owns property

### PR DESCRIPTION
**Pull request by kara**
r: @jelbourn

### CODE REVIEW

File: src/lib/select/option.ts [hunk 1]
```diff
@@ -10,13 +10,20 @@ import {
 import {ENTER, SPACE} from '../core/keyboard/keycodes';
 import {coerceBooleanProperty} from '../core/coersion/boolean-property';
 
+/**
+ * Option IDs need to be unique across components, so this counter exists outside of
+ * the component definition.
+ */
+var _selectOptionCounter = 0;
```
Comment (jelbourn) [inline src/lib/select/option.ts]: How about?
```ts
/** Unique ID counter for md-option. */
let _uniqueIdCounter = 0;
```

File: src/lib/select/select.ts [hunk 2]
```diff
@@ -291,6 +300,11 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
     this._subscriptions = [];
   }
 
+  /** Records option IDs to pass to the aria-owns property. */
+  private _setOptionIds() {
+    this._optionIds = this.options.map((option) => option.id).join(' ');
```
Comment (jelbourn) [inline src/lib/select/select.ts]: nit: don't need parens around `option`

File: src/lib/select/select.spec.ts [hunk 3]
```diff
@@ -597,6 +596,59 @@ describe('MdSelect', () => {
...
+      it('should set aria-owns properly', () => {
+        const selects = fixture.debugElement.queryAll(By.css('md-select'));
+
+        expect(selects[0].nativeElement.getAttribute('aria-owns'))
+            .toContain('md-select-option', `Expected option IDs to be set in aria-owns attr.`);
+
+        expect(selects[0].nativeElement.getAttribute('aria-owns'))
+            .not.toEqual(selects[1].nativeElement.getAttribute('aria-owns'),
+                `Expected select components to have unique option IDs in aria-owns attr.`);
+      });
```
Comment (jelbourn) [inline src/lib/select/select.spec.ts]: Need a test where you assert that the `aria-owns` contains each of the options's IDs. Something like
```
expect(ariaOwns).toContain(options[0].id);
expect(ariaOwns).toContain(options[1].id);
...
```

**Comment by kara** [comment]
@jelbourn Comments addressed!

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**